content/en/docs/tasks/tls/managing-tls-in-a-cluster.md

```diff
@@ -262,7 +262,7 @@ Finally, populate the signed certificate in the API object's status:
 
 ```shell
 kubectl get csr my-svc.my-namespace -o json | \
-  jq '.status.certificate = "'$(base64 ca-signed-server.pem | tr -d '\n')'"' | \
+  jq '.status.certificate = "'$(base64 -i ca-signed-server.pem | tr -d '\n')'"' | \
```

Comment:
With the BSD version of base64 included with macOS, `-i` means "input file". It does not take a positional input file argument.
With the gnu version in most linux distros, `-i` is short for `--ignore-garbage`. It _does_ allow for a positional input file argument.

There shouldn't be any "garbage" in the pem file, but it looks safe to use it to be compatible with either implementation.